fonts/fonts.css

```diff
@@ -0,0 +1,26 @@
+@font-face { 
```

Comment:
Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/GurevAleksey_itogovaya/pullRequest?prid=6088340)